docs/en/analytics/data-mgmt/index.md

```diff
@@ -72,4 +72,7 @@ Follow the same steps to update display name and description for Event Parameter
 3. Click on the column labeled **Display Value**, enter a new value, such as `Non-First Entry` for `0`, and enter `First Entry` for `1`, then click confirm.
 4. Go back to Exploration, and in the filter dropdown, you can see `_entrances` displayed as `Non-First Entry` and `First Entry`.
 
-Follow the same steps to customize data dictionary for User Attributes values.
\ No newline at end of file
+Follow the same steps to customize data dictionary for User Attributes values.
+
+## Trigger scan metadata workflow manually
+You can to click the **Scan metadata** button located in the top right corner of the page to start a workflow to scan the past seven days metadata of the current app. 
```

Comment:
## Trigger the workflow to scan metadata manually
You can choose **Scan metadata** at the top right corner of the page to start a workflow to scan the past seven days metadata of the current app.